gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/TemplateMapperTest.java

```diff
@@ -52,11 +53,17 @@
     private CodeableConceptMapper codeableConceptMapper;
 
     @Mock
-    private ImmunizationChecker immunizationChecker;
+    private DatabaseImmunizationChecker immunizationChecker;
 
-    @InjectMocks
+    private ResourceReferenceUtil resourceReferenceUtil;
     private TemplateMapper templateMapper;
```

Comment:
The same comment as for EncounterMapper test, you could now mock ResourceReferenceUtil (then DatabaseImmunizationChecker mock won't be needed)